### PR TITLE
Add RAM requirement

### LINKED ISSUE #1152: State the minimal requirement of RAM to build invidious in the README.md
I tried to do the setup of invidious as documented here:
https://github.com/omarroth/invidious#setup-invidious
on a virtual machine Debian 10 (buster) 32bits with 512GB of ram, it failed. 
There is a reference in #209 saying 2GB of ram is needed. That would be very nice to state it in the README.md file.
Cheers,

### PR DESCRIPTION
**Pull request by SuperSandro2000**
Closes #1152

### CODE REVIEW

**Comment by cloudrac3r** [comment]
A smaller amount of RAM also works if RAM + swap >= 2 GB. You might want to update this to reference swap.

**Comment by SuperSandro2000** [comment]
Compiling on swap is slower and everyone should have 2GB of RAM nowdays.

**Comment by cloudrac3r** [comment]
It's slower but it actually works, unlike 1 GB total.

If everyone should have 2 GB+, why is this PR necessary at all?

**Comment by SuperSandro2000** [comment]
> It's slower but it actually works, unlike 1 GB total.

Docker also works on 1GB.

> If everyone should have 2 GB+, why is this PR necessary at all?

Then close the issue that requested this.

> And slow compiling is better than no compiling :)

This does not forbid you do to that.

**Comment by gripped** [comment]
Productive would be editing the PR to state reality and not a falsehood. Some people might not know they can use swap as as workaround.
Discussing not arguing :)

File: README.md [hunk 1]
```diff
@@ -73,7 +73,9 @@ $ docker volume rm invidious_postgresdata
 $ docker-compose build
 ```
 
-### Linux:
+### Linux
+
+To manually compile invidious you need at least 2GB of RAM. If you have less you can setup SWAP to have a combined amount of 2 GB or fall back to Docker.
```
Comment (Perflyst) [inline README.md:78]:
```suggestion
To manually compile invidious you need at least 2GB of RAM. If you have less you can setup SWAP to have a combined amount of 2 GB or use Docker instead.
```

**Pull request merged**